q2_diversity/_alpha.py

```diff
@@ -5,3 +5,110 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
+
+import os
+
+import biom
+import scipy
+import skbio
+import qiime
+import pandas as pd
+import seaborn as sns
+import skbio.diversity
+from statsmodels.sandbox.stats.multicomp import multipletests
+
+
+# We should consider moving these functions to scikit-bio. They're part of
+# the private API here for now.
+def phylogenetic_metrics():
+    return {'faith_pd'}
+
+
+def non_phylogenetic_metrics():
+    return {'ace', 'chao1', 'chao1_ci', 'berger_parker_d', 'brillouin_d',
+            'dominance', 'doubles', 'enspie', 'esty_ci', 'fisher_alpha',
+            'goods_coverage', 'heip_e', 'kempton_taylor_q', 'margalef',
+            'mcintosh_d', 'mcintosh_e', 'menhinick', 'michaelis_menten_fit',
+            'observed_otus', 'osd', 'pielou_e', 'robbins', 'shannon',
+            'simpson', 'simpson_e', 'singles', 'strong', 'gini_index',
+            'lladser_pe', 'lladser_ci'}
+
+
+def alpha_phylogenetic(metric: str, feature_table: biom.Table,
```

Comment:
We're thinking the framework may enforce input artifacts come first, followed by parameters (see discussion: https://github.com/qiime2/qiime2/issues/70). Right now this works but may not in the future, so probably worth updating. @ebolyen do you agree?

I think the new signature should be:

```python
alpha_phylogenetic(feature_table: biom.Table, phylogeny: skbio.TreeNode, metric: str) -> pd.Series
```

Note that since this is a phylogenetic method, `phylogeny` should be a required param.